### DISCUSSION TITLE
Unable to react to releases

### DISCUSSION THREAD

**Discussion by amadejpapez** [community/community#68085] [Mobile] [answered]
For a couple weeks now I have been missing the button to react to releases or to even see reactions from others. I remember this was there before.

For example: https://github.com/mastodon/mastodon/releases/tag/v4.2.0

iOS app:
![image](https://github.com/community/community/assets/63184600/564fdca5-ea32-429f-a079-b4fd36388fed)

Safari:
![image](https://github.com/community/community/assets/63184600/fa6dcbf9-2945-4472-bb3c-f3f1f460de46)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.131.0 (110081196)
Device: iPhone15,5 (iOS 17.0.2)
</details>

**Comment by alcere** [comment]
Hi @amadejpapez, we are sorry for the troubles! I see the same behaviour. I will create an internal issue to fix this bug! thanks!

**Reply by jsejcksn** [reply]
Thanks @alcere. Any update to share? It's still ongoing for some months now.

**Comment by amadejpapez** [comment, marked as answer]
Seems fixed in 1.144.0 👍
